assets/src/blocks/CarouselHeader/SlidesContainer.js

```diff
@@ -1,10 +1,7 @@
 import { ArrowsAndIndicators } from './ArrowsAndIndicators';
 
-import {
-  Children,
```

Comment:
Not sure why this was used and not just a regular map. Perhaps it's somehow needed in conjunction with [`cloneElement` that's used below](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/603/files#diff-24823ce53c591e97c607cbf7001b17b9977148cfac77f9b53be687ac20c6035bL18)?

Reply:
Yeah fair point, I've reverted it for now just to be sure 🙂